googler

```diff
@@ -26,8 +26,8 @@ import struct
 import webbrowser
 import gzip
 from getopt import getopt, GetoptError
-import readline
```

Comment:
readline is necessary. Otherwise left/right arrow keys do not work in search prompt.